shared/src/utils/connectors.ts

```diff
@@ -18,8 +21,16 @@ export const getWalletConnectConnector = () =>
   new WalletConnectConnector({
     supportedChainIds,
     rpc: isDevelopment()
-      ? { 42: process.env.REACT_APP_TESTNET_URI || "" }
-      : { 1: process.env.REACT_APP_MAINNET_URI || "" },
+      ?
+        {
+          [CHAINID.ETH_KOVAN]: process.env.REACT_APP_TESTNET_URI || "",
+          [CHAINID.AVAX_FUJI]: process.env.REACT_APP_TESTNET_URI || "",
+        }
+      :
+        {
+          [CHAINID.ETH_MAINNET]: process.env.REACT_APP_MAINNET_URI || "",
+          [CHAINID.AVAX_MAINNET]: process.env.REACT_APP_MAINNET_URI || ""
```

Comment:
These will need to be set to different values though? we can split out to `REACT_APP_AVAX_URI`